templates/puppetchef/update_props.erb

```diff
@@ -0,0 +1,23 @@
+<% update_props = properties_by_custom_update(object.all_user_properties) -%>
```

Comment:
I'd like to start putting the shared Puppet+Chef code inside a `puppetchef` folder to clean up the templates/ directory. Any objections?

Reply:
SGTM.